apps/storybook/src/Dialog.stories.tsx

```diff
@@ -277,3 +277,64 @@ export const DraggableRelativeToContainer: Story = ({ isOpen, ...rest }) => {
     </div>
   );
 };
+
+export const Position: Story = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const closeDialog = () => {
+    setIsOpen(false);
+  };
+
+  const onClose = (event: React.SyntheticEvent<Element, Event>) => {
+    action('onClose', { depth: 1 })(event);
+    closeDialog();
+  };
+
+  const primaryButtonHandle = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => {
+    action('Primary button')(event);
+    closeDialog();
+  };
+
+  const secondaryButtonHandle = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => {
+    action('Secondary button')(event);
+    closeDialog();
+  };
+
+  return (
+    <>
+      <Button styleType='high-visibility' onClick={() => setIsOpen(true)}>
+        Open Dialog
+      </Button>
+      <Dialog
+        isOpen={isOpen}
+        position={'top-left'}
+        onClose={onClose}
+        closeOnEsc
+        isDismissible
```

Comment:
forward the rest args here (see lines 200 and 251). this will allow the user to change the placement using story controls

Reply:
got it